NOTICE-binary

```diff
@@ -0,0 +1,761 @@
+
+ResolverUtil.java
+Copyright 2005-2006 Tim Fennell
+
+Apache Log4j SLF4J Binding
+Copyright 1999-2022 The Apache Software Foundation
+
+Apache Ratis Client
+Copyright 2017-2020 The Apache Software Foundation
+
+Apache Ratis Common
+Copyright 2017-2020 The Apache Software Foundation
+
+Apache Ratis gRPC Support
+Copyright 2017-2020 The Apache Software Foundation
+
+Apache Ratis Metrics
+Copyright 2017-2020 The Apache Software Foundation
+
+Apache Ratis Netty Support
+Copyright 2017-2020 The Apache Software Foundation
+
+Apache Ratis Protocols
+Copyright 2017-2020 The Apache Software Foundation
+
+Apache Ratis Server
+Copyright 2017-2020 The Apache Software Foundation
+
+Apache Ratis Server API
+Copyright 2017-2020 The Apache Software Foundation
+
+Apache Ratis Thirdparty Miscellaneous
+Copyright 2017-2020 The Apache Software Foundation
+
+---
+
+This product bundles protobuf-java which includes the following:
+
+Copyright 2008 Google Inc.
+
+---
+
+This product bundles gRPC which includes the following text:
+
+Copyright 2014 The gRPC Authors
+
+---
+
+This product bundles Netty which includes the following text:
+
+Copyright 2014 The Netty Project
+
+---
+
+This product bundles Gson which includes the following text:
+
+Copyright 2008 Google Inc.
+
+The Netty Project
+                            =================
+
+Please visit the Netty web site for more information:
+
+  * http://netty.io/
+
+Copyright 2016 The Netty Project
+
+The Netty Project licenses this file to you under the Apache License,
+version 2.0 (the "License"); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at:
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+License for the specific language governing permissions and limitations
+under the License.
+
+-------------------------------------------------------------------------------
+This product contains a forked and modified version of Tomcat Native
+
+  * LICENSE:
+    * license/LICENSE.tomcat-native.txt (Apache License 2.0)
+  * HOMEPAGE:
+    * http://tomcat.apache.org/native-doc/
+    * https://svn.apache.org/repos/asf/tomcat/native/
+
+This product contains the Maven wrapper scripts from 'Maven Wrapper', that provides an easy way to ensure a user has everything necessary to run the Maven build.
+
+  * LICENSE:
+    * license/LICENSE.mvn-wrapper.txt (Apache License 2.0)
+  * HOMEPAGE:
+    * https://github.com/takari/maven-wrapper
+
+This product contains small piece of code to support AIX, taken from netbsd.
+
+  * LICENSE:
+    * license/LICENSE.aix-netbsd.txt (OpenSSL License)
+  * HOMEPAGE:
+    * https://ftp.netbsd.org/pub/NetBSD/NetBSD-current/src/crypto/external/bsd/openssl/dist
+
+
+This product contains code from boringssl.
```

Comment:
as I said previous, this NOTICE should reflect the real contains of tarball, we don't contains that, and there are some duplications which need to fix